services/cloudsecuritygroup/securitygroupazure/securitygroupazure.go

```diff
@@ -0,0 +1,153 @@
+package securitygroupazure
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/dome9/dome9-sdk-go/services/cloudsecuritygroup"
+)
+
+type CloudSecurityGroupRequest struct {
+	Name              string             `json:"name"`
+	Description       string             `json:"description"`
+	Region            string             `json:"region"`
+	IsTamperProtected bool               `json:"isTamperProtected"`
+	ResourceGroup     string             `json:"resourceGroup"`
+	CloudAccountID    string             `json:"cloudAccountId"`
+	Tags              []Tags             `json:"tags"`
+	InboundServices   []InboundServices  `json:"inboundServices"`
+	OutboundServices  []OutboundServices `json:"outboundServices"`
+}
+
+type Tags struct {
+	Key   string `json:"key"`
+	Value string `json:"value"`
+}
+
+//type Data struct {
+//	Cidr string `json:"cidr"`
+//	Note string `json:"note"`
+//}
+
+type Scope struct {
+	Type string            `json:"type"`
+	Data map[string]string `json:"data"`
+}
+
+type InboundServices struct {
+	Direction             string   `json:"direction"`
+	Name                  string   `json:"name"`
+	Description           string   `json:"description"`
+	Priority              int      `json:"priority"`
+	Access                string   `json:"access"`
+	Protocol              string   `json:"protocol"`
+	SourcePortRanges      []string `json:"sourcePortRanges"`
+	SourceScopes          []Scope  `json:"sourceScopes"`
+	DestinationPortRanges []string `json:"destinationPortRanges"`
+	DestinationScopes     []Scope  `json:"destinationScopes"`
+	IsDefault             bool     `json:"isDefault"`
+}
+
+type OutboundServices struct {
+	Direction             string   `json:"direction"`
+	Name                  string   `json:"name"`
+	Description           string   `json:"description"`
+	Priority              int      `json:"priority"`
+	Access                string   `json:"access"`
+	Protocol              string   `json:"protocol"`
+	SourcePortRanges      []string `json:"sourcePortRanges"`
+	SourceScopes          []Scope  `json:"sourceScopes"`
+	DestinationPortRanges []string `json:"destinationPortRanges"`
+	DestinationScopes     []Scope  `json:"destinationScopes"`
+	IsDefault             bool     `json:"isDefault"`
+}
+
+type CloudSecurityGroupResponse struct {
```

Comment:
Put the definition of this struct after the Request struct as we did in the rest like this order:
Request
Response
Related structs
Get
GetAll
Create
Delete
Updates